api/issues_tiles.py

```diff
@@ -191,7 +191,7 @@ def issues_mvt(db, z, x, y, format):
             response.content_type = 'application/vnd.mapbox-vector-tile'
             return tile
         else:
-            return HTTPError(404)
+            return HTTPResponse(status=204, headers={'Access-Control-Allow-Origin': '*'})
```

Comment:
The CORS is added globally, not to each request.
https://github.com/osm-fr/osmose-frontend/blob/master/modules/bottle_cors.py